linker_scripts/code_script.txt

```diff
@@ -2061,7 +2061,7 @@ SECTIONS
     ovl_En_Tubo_Trap : AT(RomLocation)
     {
         build/src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap.o(.text)
-        build/asm/overlays/ovl_En_Tubo_Trap_data.o(.data)
+        build/src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap.o(.data)
         build/src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap.o(.rodata)
         build/asm/overlays/ovl_En_Tubo_Trap_rodata.o(.rodata)
```

Comment:
Once you've migrated all the .data and .rodata, you should change this line to be
```suggestion
        build/src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap_overlay.o(.ovl)
```
This is the section that gives to relocations for the overlay.
Having `NON_MATCHING` _may_ cause this to not work, but it shouldn't. So if it gives you issues make a note of it since that probably means we need to fix things.

Reply:
Last I tried it didn't work and I assumed it was the non-matching, but I might have just attempted wrong, will re-attempt